addons/survey/static/src/scss/survey_survey_views.dark.scss

```diff
@@ -0,0 +1,7 @@
+// = Survey view
+// ============================================================================
+// No CSS hacks, variables overrides only
```

Comment:
Is it a comment, a command, or a wish ?

Reply:
If AntonyLesuisse is the committer, do you call the police ?

Reply:
/invoke MOC

Joke aside, there should be a really (really really) good reason to put anything else than a variable override in the `*.dark.scss`files... And even then, something funky is probably lurking around.